packages/playground/src/weblets/profile_manager.vue

```diff
@@ -462,6 +490,15 @@ watch(
     }
   },
 );
+watch(
+  () => keypairType.value,
+  async (value, oldValue) => {
+    if (value !== oldValue) {
+      mnemonicInput.value?.validate();
+    }
+  },
+  { deep: false },
+);
```

Comment:
can we please update the message on line 567 to "Couldn't get the user twin for the provided mnemonic **and keytype**".